draft-irtf-cfrg-hpke.md

```diff
@@ -267,7 +267,7 @@ HPKE variants rely on the following primitives:
   - Npk: The length in bytes of an encoded public key for this KEM
 
 * A Key Derivation Function (KDF):
-  - Extract(salt, IKM): Extract a pseudorandom key of fixed length
+  - Extract(salt, IKM): Extract a pseudorandom key of fixed length `Nh`
```

Comment:
maybe add bytes as we did in some other places? Like “of fixed length `Nh` bytes”, or “of length `Nh` bytes”